scripts/set1155Stages.ts

```diff
@@ -91,15 +91,20 @@ export const set1155Stages = async (
     }),
   );
 
-  const stages = stagesConfig.map((s, i) => ({
-    price: [ethers.utils.parseEther(s.price)],
-    mintFee: [s.mintFee ? ethers.utils.parseEther(s.mintFee) : 0],
-    maxStageSupply: [s.maxSupply ?? 0],
-    walletLimit: [s.walletLimit ?? 0],
-    merkleRoot: [merkleRoots[i]],
-    startTimeUnixSeconds: Math.floor(new Date(s.startDate).getTime() / 1000),
-    endTimeUnixSeconds: Math.floor(new Date(s.endDate).getTime() / 1000),
-  }));
+  const stages = stagesConfig.map((s, i) => {
+    const price = ensureArray(s.price).map(p => ethers.utils.parseEther(p));
```

Comment:
Why `price` is handled differently compared to other fiends such as `mintFee`?

Reply:
@channing-magiceden 

It's being used as source of truth (elements in price array dictates how many editions). Another more expressive way I just thought to do now is to pass in editions as a variable and validate lengths here instead of using `price`.